src/main/kotlin/org/opensearch/replication/util/ValidationUtil.kt

```diff
@@ -105,6 +105,25 @@ object ValidationUtil {
             validationException.addValidationError("Value $name must not start with '.'")
     }
 
+    /**
+     * Validate the pattern against the rules that we have for indexPattern name.
+     */
+
+    fun validatePattern(pattern: String?, validationException: ValidationException) {
```

Comment:
Let's name this as `validateAutofollowPattern`.

Reply:
Done